### PR TITLE
Feature: swipable photos

### PR DESCRIPTION
**Pull request by DrumsnChocolate**
implements #424

### CODE REVIEW

**Review by wilco375 — DISMISSED** [review]
Can you remove the console logs? Other than that it looks good and seems to be working fine.

File: app/components/swipeable.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    this.yStart = null;
+    this.xEnd = null;
+    this.yEnd = null;
+  }
+
+  @action
+  onTouchStart(touchStartEvent) {
+    if (touchStartEvent.targetTouches.length === 1) {
+      this.touching = true;
+      let touch = touchStartEvent.targetTouches.item(0);
```
Comment (guidojw) [inline app/components/swipeable.js:28]: Can you use `const` instead of `let` on places where the variables do not change?

**Review by guidojw — CHANGES_REQUESTED** [review]

File: app/components/swipeable.js (hunk 1 above)
Comment (DrumsnChocolate) [reply app/components/swipeable.js:28]: Sure, but shouldn't small things like these be caught by the linter?

**Comment by DrumsnChocolate** [comment]
Lol, I just discovered the photo album swipes the wrong way. Tested it on an album with 2 photos first, now with 3, I see that it loops in the opposite direction of what the user would expect.

**Pull request merged**